### ISSUE TITLE
gpg not found in $PATH

### ISSUE THREAD

**Issue by peterbourgon** [keybase/keybase-issues#1783]
I'm trying the new kbstage. It's not detecting gpg in my $PATH.

```
ugh ~ kbstage login
▶ WARNING Running in staging mode
keybase passphrase:
Enter a public name for this device: ugh
How would you like to sign this install of Keybase?

(1) using PGP                    (requires access to your PGP key)
(999) Cancel, I'll login later   (cancel)
Choose a signing option: 1
▶ INFO deviceSignPGP error: exec: "gpg": executable file not found in $PATH
How would you like to sign this install of Keybase?

(1) using PGP                    (requires access to your PGP key)
(999) Cancel, I'll login later   (cancel)
Choose a signing option: 999
▶ ERROR cancel requested by user
ugh ~ which gpg
/usr/local/bin/gpg
ugh ~ echo $PATH
/Users/peter/bin /usr/local/go/bin /usr/local/bin /usr/bin /bin /usr/local/sbin /usr/sbin /sbin
ugh ~
```

**Comment by peterbourgon** [comment]
I can't seem to attach labels to this issue. Sorry!

**Comment by Sidnicious** [comment]
Is your $PATH space separated?

**Comment by peterbourgon** [comment]
That's just how [fish](http://fishshell.com/) works.

**Comment by cjb** [comment]
@peterbourgon Please could you run again with "kbstage -d login" (which turns on debug logs) and email the output to cjb@keyba.se?  That should tell us which path we tried to execute gpg at.

(Asking for an email due to the slim possibility that there'll be something secret in the debug logs for some reason.)

**Comment by patrickxb** [comment]
@peterbourgon We have a fix that will let you login with kbstage.  Will release updated build tomorrow morning.

Thanks!

**Comment by maxtaco** [comment]
And in the mean time, @peterbourgon, we have a related server fix that just went live.  Either the server or the client fix will solve this issue, so maybe try again? Thanks!

**Comment by peterbourgon** [comment]
Yep, a `kbstage login` this morning appeared to work. Thanks y'all.

Another thing: I fat-fingered my passphrase and did a (readline-standard) ctrl-U to retry. But it looks like your passphrase input package doesn't support ctrl-U? Is that deliberate for e.g. security?

**Comment by maxtaco** [comment]
Hm, we're using `gopass` which uses `crypto/ssh/terminal`. It's likely a shortcoming of the terminal implementation.  Thanks for the great feedback!
